packages/website/src/components/team/TeamBio.module.css

```
@@ -0,0 +1,47 @@
+.teamBio {
+  --square-size: 9rem;
+
+  display: flex;
+  flex-direction: column;
+  max-width: var(--square-size);
+}
+
+.profilePhoto {
+  width: var(--square-size);
+  height: var(--square-size);
+}
+
+.texts {
+  display: flex;
+  gap: 0.5rem;
+  margin-top: 0.75rem;
+  flex-direction: column;
+}
+
+.name {
+  font-size: 0.9rem;
+}
+
+.description {
+  font-size: 0.85rem;
+}
+
+.services {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 0.5rem;
+  list-style-type: none;
+  padding-left: 0;
+}
+
+.serviceIcon {
+  --size: 1.25rem;
+
+  height: var(--size);
+  transition: opacity 200ms;
+  width: var(--size);
```

[Very Nit]

Put `height` & `width` together?

```suggestion
  width: var(--size);
  height: var(--size);
  transition: opacity 200ms;
```

Or maybe even use `aspect-ratio`?

```suggestion
  width: var(--size);
  aspect-ratio: 1;
  transition: opacity 200ms;
```